app/src/pages/inside/common/assignments/instanceAssignment/instanceAssignment.tsx

```diff
@@ -113,6 +113,7 @@ interface InstanceAssignmentItem {
 interface InstanceAssignmentProps extends MyFieldArrayProps<InstanceAssignmentItem> {
   formName: string;
   formNamespace?: string;
+  isOrganizationRequired: boolean;
 }
 
 interface MyFieldArrayProps<T> extends WrappedFieldArrayProps<T> {
```

Comment:
I noticed that we have a mock naming here for `MyFieldArrayProps `, could you please update it as you already working in this file?